data/json/items/classes/magazine.json

```diff
@@ -12,6 +12,7 @@
     "color": "ltgray",
     "rigid": false,
     "reliability": 6,
+    "capacity": 9999,
```

Comment:
This isn't an intrinsic property of ammo belts and more correctly belongs in the individual definition of each item. Also 10'000 rounds is both unrealistic and going to result in obvious exploits considering you can wear ammo belts